TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift

```diff
@@ -55,7 +55,7 @@ struct KeychainQuery: KeychainQueryProtocol {
 
   /// Direct use of [kSecAttrAccessGroup](https://developer.apple.com/documentation/security/ksecattraccessgroup?language=swift),
   /// when set all methods will make use of it.
-  var accessGroup: String?
+  let accessGroup: String?
 
   init(accessGroup: String?) {
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/a8a0ffb2344905f9a27850246dcbbe000db39834/TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift#L60"></span>This memberwise initializer would be synthesized automatically - you do not need to define it<br />
<code>unneeded_synthesized_initializer</code> <code>KeychainQuery.swift:60</code></td>
    </tr>
  </tbody>
</table>